test/functional/feature_proxy.py

```diff
@@ -169,33 +170,38 @@ def networks_dict(d):
                 r[x['name']] = x
             return r
 
-        # test RPC getnetworkinfo
+        self.log.info("Test RPC getnetworkinfo")
         n0 = networks_dict(self.nodes[0].getnetworkinfo())
-        for net in ['ipv4','ipv6','onion']:
+        assert_equal(NETWORKS, n0.keys())
+        for net in NETWORKS:
             assert_equal(n0[net]['proxy'], '%s:%i' % (self.conf1.addr))
             assert_equal(n0[net]['proxy_randomize_credentials'], True)
         assert_equal(n0['onion']['reachable'], True)
 
         n1 = networks_dict(self.nodes[1].getnetworkinfo())
-        for net in ['ipv4','ipv6']:
+        assert_equal(NETWORKS, n1.keys())
+        for net in ['ipv4', 'ipv6']:
             assert_equal(n1[net]['proxy'], '%s:%i' % (self.conf1.addr))
             assert_equal(n1[net]['proxy_randomize_credentials'], False)
         assert_equal(n1['onion']['proxy'], '%s:%i' % (self.conf2.addr))
         assert_equal(n1['onion']['proxy_randomize_credentials'], False)
         assert_equal(n1['onion']['reachable'], True)
 
         n2 = networks_dict(self.nodes[2].getnetworkinfo())
-        for net in ['ipv4','ipv6','onion']:
+        assert_equal(NETWORKS, n2.keys())
+        for net in NETWORKS:
             assert_equal(n2[net]['proxy'], '%s:%i' % (self.conf2.addr))
             assert_equal(n2[net]['proxy_randomize_credentials'], True)
         assert_equal(n2['onion']['reachable'], True)
 
         if self.have_ipv6:
             n3 = networks_dict(self.nodes[3].getnetworkinfo())
-            for net in ['ipv4','ipv6']:
+            assert_equal(NETWORKS, n3.keys())
+            for net in NETWORKS:
```

Comment:
This used to run for just `'ipv4','ipv6'` before and now will also run for `'onion'` in addition. I guess it is ok as long as the test passes.

Reply:
Thanks. Yes, that was my thinking: if it passes with the additional coverage, we'll take it.